home/src/main/resources/rdf/i18n/en_US/interface-i18n/firsttime/vitro_extended_search.ttl

```diff
@@ -0,0 +1,48 @@
+@prefix owl:   <http://www.w3.org/2002/07/owl#> .
+@prefix rdf:   <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix prop-data: <http://vivoweb.org/ontology/core/properties/individual#> .
+@prefix prop:  <http://vivoweb.org/ontology/core/properties/vocabulary#> .
+@prefix xsd:   <http://www.w3.org/2001/XMLSchema#> .
+@prefix skos:  <http://www.w3.org/2004/02/skos/core#> .
+@prefix rdfs:  <http://www.w3.org/2000/01/rdf-schema#> .
+
+prop-data:search_results_relevance.Vitro
```

Comment:
Why those UI labels are in separated file? Actually, the question should be why all others UI labels are in one file :-) I think this is a nice practice to decouple UI labels per functionalities, although it might be more complicate for someone who would like to translate UI to new language

Reply:
All those labels are used on extended search results page as I remember.